app/redux/slices/exchangeSlice18.ts

```diff
@@ -0,0 +1,36 @@
+import { createSlice } from '@reduxjs/toolkit';
+import { IApiCourse } from '../../models/ICourse';
+import { IExchange } from '../../services/api/api-response.types';
+
+interface ICourseState {
+  course: {
+    exchangeRate: IApiCourse[];
+    date: string;
+    bank: string;
+    baseCurrency: number;
+    baseCurrencyLit: string;
+  } | null;
```

Comment:
Create a type for this inside `types` dir or in somewhere else.
(Also you can check `IApiCourse` type, maybe it does match)